app/wama.web/Views/Patron/Create.cshtml

```diff
@@ -13,8 +13,6 @@
             <label asp-for="MemberId"> </label>
             <input asp-for="MemberId"
                    class="form-control"
-                   pattern="[0-9]{7}"
-                   title="Member ID must be in number and 7 digit long."
```

Comment:
Yep, that's true.

But you're still not seeing/understanding how these validations work.
Or, why they're needed.
Think of, and answer these questions.

* Where should the validation happen if I use a browser that doesn't support HTML5?
* Where should the validation happen if I use a browser that supports HTML5?
* Should the request always have to go back to the server in other for validation to happen?